### ISSUE TITLE
Importing Error on PostgreSQL 9.6

### ISSUE THREAD

**Issue by cmennens** [devrimgunduz/pagila#3]
When you run the following command as postgres user on v9.6, I get the following errors:

```
psql pagila < pagila-schema.sql
[...]
ALTER TABLE
ALTER TABLE
ERROR:  relation "payment_p2017_01" does not exist
ERROR:  relation "payment_p2017_02" does not exist
ERROR:  relation "payment_p2017_03" does not exist
ERROR:  relation "payment_p2017_04" does not exist
ERROR:  relation "payment_p2017_05" does not exist
ERROR:  relation "payment_p2017_06" does not exist
```

This error is present on CentOS / Debian / Arch / Ubuntu.

**Comment by bmintz** [comment]
If you run `psql pagila -f pagila-schema.sql` instead, line numbers will be output for each error. What do you get?

**Comment by Aldenar** [comment]
Can confirm I have the same issue on Postgres 9.6.15. 

My database was created like so:
`/usr/bin/createdb --template=template0 --encoding=UTF8 --lc-collate=cs_CZ.utf8 --lc-ctype=cs_CZ.utf8 --owner=postgres test_db`

Following that up with a grant of
`GRANT ALL PRIVILEGES ON DATABASE test_db TO test_user;`

Running as said non-superuser account produces the following:

```
psql -U test_user test_db -f pagila-schema.sql
psql:pagila-schema.sql:28: ERROR:  must be owner of extension plpgsql
psql:pagila-schema.sql:46: ERROR:  must be member of role "postgres"
psql:pagila-schema.sql:56: ERROR:  must be member of role "postgres"
psql:pagila-schema.sql:73: ERROR:  must be member of role "postgres"
psql:pagila-schema.sql:90: ERROR:  must be member of role "postgres"
psql:pagila-schema.sql:107: ERROR:  must be member of role "postgres"
psql:pagila-schema.sql:152: ERROR:  must be member of role "postgres"
psql:pagila-schema.sql:174: ERROR:  must be member of role "postgres"
psql:pagila-schema.sql:211: ERROR:  must be member of role "postgres"
psql:pagila-schema.sql:229: ERROR:  must be member of role "postgres"
psql:pagila-schema.sql:244: ERROR:  must be member of role "postgres"
psql:pagila-schema.sql:258: ERROR:  must be member of role "postgres"
psql:pagila-schema.sql:282: ERROR:  must be member of role "postgres"
psql:pagila-schema.sql:346: ERROR:  must be member of role "postgres"
psql:pagila-schema.sql:358: ERROR:  must be member of role "postgres"
psql:pagila-schema.sql:372: ERROR:  must be member of role "postgres"
psql:pagila-schema.sql:386: ERROR:  must be member of role "postgres"
psql:pagila-schema.sql:400: ERROR:  must be member of role "postgres"
psql:pagila-schema.sql:413: ERROR:  must be member of role "postgres"
psql:pagila-schema.sql:427: ERROR:  must be member of role "postgres"
psql:pagila-schema.sql:451: ERROR:  must be member of role "postgres"
psql:pagila-schema.sql:464: ERROR:  must be member of role "postgres"
psql:pagila-schema.sql:477: ERROR:  must be member of role "postgres"
psql:pagila-schema.sql:500: ERROR:  must be member of role "postgres"
psql:pagila-schema.sql:514: ERROR:  must be member of role "postgres"
psql:pagila-schema.sql:532: ERROR:  must be member of role "postgres"
psql:pagila-schema.sql:546: ERROR:  must be member of role "postgres"
psql:pagila-schema.sql:560: ERROR:  must be member of role "postgres"
psql:pagila-schema.sql:574: ERROR:  must be member of role "postgres"
psql:pagila-schema.sql:587: ERROR:  must be member of role "postgres"
psql:pagila-schema.sql:612: ERROR:  must be member of role "postgres"
psql:pagila-schema.sql:635: ERROR:  must be member of role "postgres"
psql:pagila-schema.sql:649: ERROR:  must be member of role "postgres"
psql:pagila-schema.sql:663: ERROR:  must be member of role "postgres"
psql:pagila-schema.sql:677: ERROR:  must be member of role "postgres"
psql:pagila-schema.sql:690: ERROR:  must be member of role "postgres"
psql:pagila-schema.sql:713: ERROR:  must be member of role "postgres"
psql:pagila-schema.sql:727: ERROR:  must be member of role "postgres"
psql:pagila-schema.sql:740: ERROR:  syntax error at or near "PARTITION"
LINE 8: ) PARTITION BY RANGE(payment_date);
          ^
psql:pagila-schema.sql:742: ERROR:  relation "payment" does not exist
psql:pagila-schema.sql:749: ERROR:  syntax error at or near "PARTITION"
LINE 1: CREATE TABLE payment_p2017_01 PARTITION OF payment
                                      ^
psql:pagila-schema.sql:751: ERROR:  relation "payment_p2017_01" does not exist
psql:pagila-schema.sql:758: ERROR:  syntax error at or near "PARTITION"
LINE 1: CREATE TABLE payment_p2017_02 PARTITION OF payment
                                      ^
psql:pagila-schema.sql:760: ERROR:  relation "payment_p2017_02" does not exist
psql:pagila-schema.sql:767: ERROR:  syntax error at or near "PARTITION"
LINE 1: CREATE TABLE payment_p2017_03 PARTITION OF payment
                                      ^
psql:pagila-schema.sql:770: ERROR:  relation "payment_p2017_03" does not exist
psql:pagila-schema.sql:777: ERROR:  syntax error at or near "PARTITION"
LINE 1: CREATE TABLE payment_p2017_04 PARTITION OF payment
                                      ^
psql:pagila-schema.sql:779: ERROR:  relation "payment_p2017_04" does not exist
psql:pagila-schema.sql:786: ERROR:  syntax error at or near "PARTITION"
LINE 1: CREATE TABLE payment_p2017_05 PARTITION OF payment
                                      ^
psql:pagila-schema.sql:788: ERROR:  relation "payment_p2017_05" does not exist
psql:pagila-schema.sql:795: ERROR:  syntax error at or near "PARTITION"
LINE 1: CREATE TABLE payment_p2017_06 PARTITION OF payment
                                      ^
psql:pagila-schema.sql:798: ERROR:  relation "payment_p2017_06" does not exist
psql:pagila-schema.sql:812: ERROR:  must be member of role "postgres"
psql:pagila-schema.sql:829: ERROR:  must be member of role "postgres"
psql:pagila-schema.sql:845: ERROR:  relation "payment" does not exist
LINE 4:    FROM (((((payment p
                     ^
psql:pagila-schema.sql:848: ERROR:  relation "sales_by_film_category" does not exist
psql:pagila-schema.sql:862: ERROR:  must be member of role "postgres"
psql:pagila-schema.sql:883: ERROR:  must be member of role "postgres"
psql:pagila-schema.sql:897: ERROR:  must be member of role "postgres"
psql:pagila-schema.sql:911: ERROR:  must be member of role "postgres"
psql:pagila-schema.sql:930: ERROR:  relation "payment" does not exist
LINE 5:    FROM (((((((payment p
                       ^
psql:pagila-schema.sql:933: ERROR:  relation "sales_by_store" does not exist
psql:pagila-schema.sql:954: ERROR:  must be member of role "postgres"
psql:pagila-schema.sql:1100: ERROR:  relation "payment" does not exist
psql:pagila-schema.sql:1135: ERROR:  relation "payment_p2017_01" does not exist
psql:pagila-schema.sql:1142: ERROR:  relation "payment_p2017_01" does not exist
psql:pagila-schema.sql:1149: ERROR:  relation "payment_p2017_02" does not exist
psql:pagila-schema.sql:1156: ERROR:  relation "payment_p2017_02" does not exist
psql:pagila-schema.sql:1163: ERROR:  relation "payment_p2017_03" does not exist
psql:pagila-schema.sql:1170: ERROR:  relation "payment_p2017_03" does not exist
psql:pagila-schema.sql:1177: ERROR:  relation "payment_p2017_04" does not exist
psql:pagila-schema.sql:1184: ERROR:  relation "payment_p2017_04" does not exist
psql:pagila-schema.sql:1191: ERROR:  relation "payment_p2017_05" does not exist
psql:pagila-schema.sql:1198: ERROR:  relation "payment_p2017_05" does not exist
psql:pagila-schema.sql:1205: ERROR:  relation "payment_p2017_06" does not exist
psql:pagila-schema.sql:1212: ERROR:  relation "payment_p2017_06" does not exist
psql:pagila-schema.sql:1219: ERROR:  relation "payment" does not exist
psql:pagila-schema.sql:1519: ERROR:  relation "payment_p2017_01" does not exist
psql:pagila-schema.sql:1524: ERROR:  relation "payment_p2017_02" does not exist
psql:pagila-schema.sql:1529: ERROR:  relation "payment_p2017_03" does not exist
psql:pagila-schema.sql:1534: ERROR:  relation "payment_p2017_04" does not exist
psql:pagila-schema.sql:1539: ERROR:  relation "payment_p2017_05" does not exist
psql:pagila-schema.sql:1544: ERROR:  relation "payment_p2017_06" does not exist
```

Running the schema file as superuser doesn't solve all the issues either, only skips the extension errors:


```
psql -U postgres test_db -f pagila-schema.sql

psql:pagila-schema.sql:740: ERROR:  syntax error at or near "PARTITION"
LINE 8: ) PARTITION BY RANGE(payment_date);
          ^
psql:pagila-schema.sql:742: ERROR:  relation "payment" does not exist
psql:pagila-schema.sql:749: ERROR:  syntax error at or near "PARTITION"
LINE 1: CREATE TABLE payment_p2017_01 PARTITION OF payment
                                      ^
psql:pagila-schema.sql:751: ERROR:  relation "payment_p2017_01" does not exist
psql:pagila-schema.sql:758: ERROR:  syntax error at or near "PARTITION"
LINE 1: CREATE TABLE payment_p2017_02 PARTITION OF payment
                                      ^
psql:pagila-schema.sql:760: ERROR:  relation "payment_p2017_02" does not exist
psql:pagila-schema.sql:767: ERROR:  syntax error at or near "PARTITION"
LINE 1: CREATE TABLE payment_p2017_03 PARTITION OF payment
                                      ^
psql:pagila-schema.sql:770: ERROR:  relation "payment_p2017_03" does not exist
psql:pagila-schema.sql:777: ERROR:  syntax error at or near "PARTITION"
LINE 1: CREATE TABLE payment_p2017_04 PARTITION OF payment
                                      ^
psql:pagila-schema.sql:779: ERROR:  relation "payment_p2017_04" does not exist
psql:pagila-schema.sql:786: ERROR:  syntax error at or near "PARTITION"
LINE 1: CREATE TABLE payment_p2017_05 PARTITION OF payment
                                      ^
psql:pagila-schema.sql:788: ERROR:  relation "payment_p2017_05" does not exist
psql:pagila-schema.sql:795: ERROR:  syntax error at or near "PARTITION"
LINE 1: CREATE TABLE payment_p2017_06 PARTITION OF payment
                                      ^
psql:pagila-schema.sql:798: ERROR:  relation "payment_p2017_06" does not exist
psql:pagila-schema.sql:845: ERROR:  relation "payment" does not exist
LINE 4:    FROM (((((payment p
                     ^
psql:pagila-schema.sql:848: ERROR:  relation "sales_by_film_category" does not exist
psql:pagila-schema.sql:930: ERROR:  relation "payment" does not exist
LINE 5:    FROM (((((((payment p
                       ^
psql:pagila-schema.sql:933: ERROR:  relation "sales_by_store" does not exist
psql:pagila-schema.sql:1100: ERROR:  relation "payment" does not exist
psql:pagila-schema.sql:1135: ERROR:  relation "payment_p2017_01" does not exist
psql:pagila-schema.sql:1142: ERROR:  relation "payment_p2017_01" does not exist
psql:pagila-schema.sql:1149: ERROR:  relation "payment_p2017_02" does not exist
psql:pagila-schema.sql:1156: ERROR:  relation "payment_p2017_02" does not exist
psql:pagila-schema.sql:1163: ERROR:  relation "payment_p2017_03" does not exist
psql:pagila-schema.sql:1170: ERROR:  relation "payment_p2017_03" does not exist
psql:pagila-schema.sql:1177: ERROR:  relation "payment_p2017_04" does not exist
psql:pagila-schema.sql:1184: ERROR:  relation "payment_p2017_04" does not exist
psql:pagila-schema.sql:1191: ERROR:  relation "payment_p2017_05" does not exist
psql:pagila-schema.sql:1198: ERROR:  relation "payment_p2017_05" does not exist
psql:pagila-schema.sql:1205: ERROR:  relation "payment_p2017_06" does not exist
psql:pagila-schema.sql:1212: ERROR:  relation "payment_p2017_06" does not exist
psql:pagila-schema.sql:1219: ERROR:  relation "payment" does not exist
psql:pagila-schema.sql:1519: ERROR:  relation "payment_p2017_01" does not exist
psql:pagila-schema.sql:1524: ERROR:  relation "payment_p2017_02" does not exist
psql:pagila-schema.sql:1529: ERROR:  relation "payment_p2017_03" does not exist
psql:pagila-schema.sql:1534: ERROR:  relation "payment_p2017_04" does not exist
psql:pagila-schema.sql:1539: ERROR:  relation "payment_p2017_05" does not exist
psql:pagila-schema.sql:1544: ERROR:  relation "payment_p2017_06" does not exist
```

These errors persist even if the target database is owned directly by the test_user account and the import is ran as superuser/normal user.

**Comment by devrimgunduz-edb** [comment]
Hi,

Should be fixed in 2.0 version.

Regards, Devrim

**Comment by devrimgunduz** [comment]
Hi,

Already fixed in 2.0

Regards, Devrrim

**Comment by devrimgunduz** [comment]
The new pagila works only against PostgreSQL 11+.

Regards, Devrim
